src/modulefinders/linux_modulefinder.cpp

```diff
@@ -63,8 +63,9 @@ int dl_iterate_callback(struct dl_phdr_info *dl_info, size_t size, void *data) {
                 offset += (size_t)nhdr->n_descsz;
                 align(alignment, &offset);
                 if (nhdr->n_type == NT_GNU_BUILD_ID) {
-                    module.set_by_key(
-                        "code_id", Value::new_hexstring(note, nhdr->n_descsz));
+                    Value code_id = Value::new_hexstring(note, nhdr->n_descsz);
+                    module.set_by_key("code_id", code_id);
+                    module.set_by_key("debug_id", code_id);
```

Comment:
The code id is most definitely not a valid debug id. Rather leave it empty, or run the correct conversion.
I believe that Symbolicator should be able to infer the debug id from the code id. If not, we can add that.